CONTRIBUTING.md

```diff
@@ -1,41 +1,58 @@
-# Contributing
+# How to contribute
```

Comment:
Let's keep `Contributing` for consistency with other repos.

Reply:
Aren't we standardizing to `How to contribute` 
https://github.com/DataDog/datadogpy/pull/503/files#diff-6a3371457528722a734f3c51d9238c13R1

Reply:
nevermind 🤦‍♂